### PR TITLE
Support API create page improvement

### PR DESCRIPTION
**Pull request by thehenster**
A helper to access the new endpoint on the Support API added here: https://github.com/alphagov/support-api/pull/96

### CODE REVIEW

File: lib/gds_api/support_api.rb [hunk 1]
```diff
@@ -21,6 +21,10 @@ def create_global_export_request(request_details)
     post_json!("#{endpoint}/anonymous-feedback/global-export-requests", global_export_request: request_details)
   end
 
+  def create_page_improvement(params)
```
Comment (Davidslv) [inline lib/gds_api/support_api.rb]: Can you please document it?
Example: https://github.com/alphagov/gds-api-adapters/pull/581/files#diff-c279de60f2b72b505bbabe7401804a88R98

Comment (thehenster) [inline lib/gds_api/support_api.rb]: Thanks. I've added some documentation of the same style.

File: lib/gds_api/support_api.rb [hunk 2]
```diff
@@ -21,6 +21,26 @@ def create_global_export_request(request_details)
     post_json!("#{endpoint}/anonymous-feedback/global-export-requests", global_export_request: request_details)
   end
 
+  # Create a Page Improvement
+  #
+  # Makes a +POST+ request to the support api to create a Page Improvement.
+  #
+  # @param params [Hash] Any attributes that relate to a Page Improvement.
```
Comment (Davidslv) [inline lib/gds_api/support_api.rb]: YARD seems to accept the type before the name of the attribute (`@param [Hash] params description` - http://www.yardoc.org/features.html)

Comment (thehenster) [inline lib/gds_api/support_api.rb]: It looks like the type can be before or after the arguments. The docs generate correctly. As the rest of the YARD annotations in this repo have the arguments first I favour keeping it this way. Are you looking for a change here?

**Pull request merged**